packages/component-survey/client/pages/index.js

```diff
@@ -1,5 +1,46 @@
-import React from 'react'
+import React, { useState } from 'react'
+import {
+  FormH2,
+  Paragraph,
+} from '@elifesciences/component-elife-ui/client/atoms'
+import { Button, TextField } from '@pubsweet/ui'
 
-const SurveyPage = () => <div>Hello Im a Survey :D</div>
+const SurveyPage = () => {
+  const [country, setCountry] = useState('')
+  const [gender, setGender] = useState('')
+  const [timeAsPI, setTimeAsPI] = useState('')
+  return (
+    <React.Fragment>
+      <FormH2>Help us reduce bias</FormH2>
+      <Paragraph.Writing>
+        Answering the following questions about the last author of your
+        submission (i.e. group leader or principal investigator) will help our
+        efforets to reduce bias in our submission and peer review process.
+      </Paragraph.Writing>
+      <Paragraph.Writing>
+        Your answers will not be shared with the Editors handling the papaer and
+        will be anonymised when we analyse the data we collect.
+      </Paragraph.Writing>
+      <TextField
+        label="Country of residence of Last Author"
+        onChange={e => setCountry(e.value)}
```

Comment:
does this need to be `e.target.value`?

Reply:
yes, e has no value so it just gets set to undefined